docs/qmk_custom_light.css

```
@@ -28,3 +28,16 @@
 .markdown-section pre {
   padding: 0;
 }
+
+
+:root {
+  --docsifytabs-border-color: #ddd;
+  --docsifytabs-tab-highlight-color: var(--theme-color, #0074d9);
+
+  --docsifytabs-tab-background: #f8f8f8;
+  --docsifytabs-tab-background-active: transparent;
+}
+
+.docsify-tabs__tab:focus {
+  outline: none !important;
+}
```

```suggestion
}

.docsify-tabs__content .anchor {
  transition: none;
}
```
https://github.com/jhildenbiddle/docsify-tabs/issues/27